code/modules/modular_computers/file_system/program.dm

```diff
@@ -244,3 +244,10 @@
 // Is called when program service is being deactivated
 /datum/computer_file/program/proc/service_deactivate()
 	return
+
+/datum/computer_file/program/proc/message_dead(var/message)
+	for(var/mob/M in player_list)
+		if(M.stat == DEAD && M.client?.prefs.toggles & CHAT_GHOSTEARS)
```

Comment:
`M.client?.prefs.toggles & CHAT_GHOSTEARS`

This will runtime if the client is null, since `null & 1` is a runtime.